### PR TITLE
Fix documentation link for Google integration

### PR DESCRIPTION
**Pull request by jorisdrenth**
### Changes

On the Search Console settings page of a selfhosted instance, the following notification is shown:

_An extra step is needed to set up your Plausible Analytics Self Hosted for the Google Search Console integration. Find instructions here_

The "here" currently leads to [this link](https://plausible.io/docs/self-hosting-configuration#google-search-integration), which does not offer relevant information about setting up Google Search Console integration on a selfhosted environment anymore. I've found the right information on the Github page of the community edition repository, so I changed the link to [the repo readme](https://github.com/plausible/community-edition/?tab=readme-ov-file#google-integration).

### Tests
- [ ] Automated tests have been added
- [x] This PR does not require tests

### Changelog
- [ ] Entry has been added to changelog
- [x] This PR does not make a user-facing change

### Documentation
- [ ] [Docs](https://github.com/plausible/docs) have been updated
- [x] This change does not need a documentation update

### Dark mode
- [x] The UI has been tested both in dark and light mode
- [ ] This PR does not change the UI

### CODE REVIEW

**Comment by jorisdrenth** [comment]
> The following files are not formatted:
> /home/runner/work/analytics/analytics/lib/plausible_web/templates/site/settings_search_console.html.heex

No idea why this is seen as not formatted and how to fix it, I'm not familiar with Elixir at all.

**Comment by ruslandoga** [comment]
👋 @jorisdrenth 

Thank you for the fix. I think the formatter wants the new longer string to go to the next line. However, I think we can use https://github.com/plausible/community-edition#google-integration as the href and it would probably be fine with the formatter to stay on the same line since it's a bit shorter.

File: lib/plausible_web/templates/site/settings_search_console.html.heex [hunk 1]
```diff
@@ -118,7 +118,7 @@
       <p class="text-gray-900 dark:text-gray-200">
         An extra step is needed to set up your <%= Plausible.product_name() %> for the Google Search Console integration.
         Find instructions <%= link("here",
-          to: "https://plausible.io/docs/self-hosting-configuration#google-search-integration",
+          to: "https://github.com/plausible/community-edition/?tab=readme-ov-file#google-integration",
```
Comment (ruslandoga) [inline lib/plausible_web/templates/site/settings_search_console.html.heex:121]:
```suggestion
          to: "https://github.com/plausible/community-edition#google-integration",
```

**Review by ruslandoga — APPROVED** [review]

**Review by zoldar — APPROVED** [review]
❤️

**Pull request merged**